src/fork_gc.c

```diff
@@ -1437,6 +1439,7 @@ void FGC_Apply(ForkGC *gc) NO_TSAN_CHECK {
 
 static void onTerminateCb(void *privdata) {
   ForkGC *gc = privdata;
+  IndexsGlobalStats_UpdateLogicallyDeleted(-gc->deletedDocsFromLastRun);
```

Comment:
is there a scenario where the gc is manually stopped and this callback will not be called?

Reply:
Shouldn't be. If so, and the spec is dropped, then no one frees the GC and we get a leak